deps/zlib/BUILD.gn

```diff
@@ -14,6 +20,10 @@ if (current_cpu == "arm" || current_cpu == "arm64") {
 
 config("zlib_config") {
   include_dirs = [ "." ]
+
+  if (zlib_symbols_visible) {
+    defines = [ "ZLIB_DLL" ]
+  }
```

Comment:
Is this something we need to add/define in zlib.gyp?

Reply:
(See the comment in lines 8-9 of this file referring to Node.js.

Reply:
Added in ab412ab2d75afc24855b5570156ab98618c6dbb8.